pep-9999.rst

```diff
@@ -0,0 +1,173 @@
+PEP: 9999
+Title: Format Specifier for Signed Zero
+Author: John Belmonte <john@neggie.net>
+Sponsor: Mark Dickinson
+Discussions-To: https://bugs.python.org/issue45995
```

Comment:
This should be a mailing list (probably python-dev; example: PEP 677) or Discuss thread (example: PEP 680).

Reply:
If it's OK, I'll merge with Discussions-To as blank and fill it in with a follow-up PR after the PEP is posted.

Reply:
+1 for using https://discuss.python.org/c/peps/ for the discussion; adding a link to the thread in a follow-up PR sounds fine to me